app/assets/images/staggered-dev.svg

```diff
@@ -0,0 +1 @@
+<svg width="106" height="103" fill="none" xmlns="http://www.w3.org/2000/svg"><g clip-path="url(#clip0)"><rect x="64.309" y="8" width="48" height="48" rx="3" transform="rotate(30.427 64.309 8)" fill="#1AB3A6"/><path d="M65.983 40.885c-.663 1.13-1.574 1.845-2.732 2.146-1.148.299-2.312.101-3.494-.593l-3.796-2.23 7.963-13.557 3.873 2.275c1.13.664 1.842 1.573 2.137 2.727.305 1.151.125 2.292-.538 3.422l-3.413 5.81zm-2.42-1.422l3.412-5.81c.284-.484.335-.94.153-1.369-.179-.435-.43-.747-.752-.937l-1.453-.853-5.119 8.715 1.453.854c.322.19.715.26 1.178.21.467-.056.843-.326 1.127-.81zm7.98 9.897l-5.084-2.986c-.484-.284-.793-.683-.926-1.195s-.058-1.01.227-1.495l5.971-10.168c.285-.484.683-.793 1.196-.926.512-.133 1.01-.057 1.494.227l5.084 2.986-1.421 2.42-4.358-2.559-1.849 3.148 2.663 1.564-1.422 2.42-2.663-1.563-1.848 3.147 4.358 2.56-1.422 2.42zm6.14 3.606c-.807-.474-1.121-1.418-.942-2.832l3.82-13.71 2.663 1.563-3.05 10.738 7.892-7.894 2.663 1.564L80.615 52.41c-1.148.846-2.125 1.031-2.932.557z" fill="#fff"/><rect x="2.491" y="59.013" width="48" height="48" rx="3" transform="rotate(-32.667 2.491 59.013)" fill="#08090A"/><path d="M32.575 72.4c.707 1.104.932 2.24.677 3.408-.253 1.159-.957 2.108-2.11 2.847l-3.706 2.377-8.486-13.236 3.781-2.425c1.103-.707 2.236-.93 3.398-.671 1.165.25 2.101.925 2.808 2.028l3.637 5.673zm-2.364 1.516l-3.637-5.672c-.303-.473-.687-.725-1.152-.756-.468-.038-.86.045-1.175.247l-1.419.909 5.456 8.509 1.418-.91c.315-.202.556-.52.721-.956.162-.441.091-.898-.212-1.37zm12.437-2.638l-4.964 3.183c-.472.303-.967.398-1.484.285-.518-.114-.928-.406-1.23-.88l-6.365-9.926c-.304-.473-.399-.968-.285-1.485.113-.517.405-.927.878-1.23l4.963-3.183 1.516 2.364-4.254 2.728 1.97 3.072 2.6-1.667 1.515 2.364-2.6 1.667 1.97 3.072 4.255-2.727 1.515 2.363zm5.994-3.843c-.788.505-1.772.358-2.952-.442l-10.498-9.611 2.6-1.667 8.195 7.579-3.468-10.61 2.6-1.667 4.353 13.551c.235 1.406-.042 2.362-.83 2.867z" fill="#fff"/></g><defs><clipPath id="clip0"><path fill="#fff" d="M0 0h106v103H0z"/></clipPath></defs></svg>
```

Comment:
I think we should also make sure to format this correctly (svgo sadly screws up formatting which makes the svg elements hard to read!)

Reply:
There's a neat flag for this, `svgo file.svg --pretty`